### PR TITLE
#158487957 Integrate HoundCI with badge attached to readme

### PR DESCRIPTION
**Pull request by nouwatinjacob**
#### What does this PR do?
- It integrates houndci and add badge to reame
#### Description of Task to be completed?
- It works with eslint to check if the pull request code conform to eslint rules
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#158487957
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```
Comment (andela-moe) [inline .hound.yml]: You should have a new line at the end of this file

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+[![Continuous Integration](https://camo.githubusercontent.com/23ee7a697b291798079e258bbc25434c4fac4f8b/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f50726f7465637465645f62792d486f756e642d6138373364312e737667)](https://houndci.com)
+
```
Comment (andela-moe) [inline README.md:2]: Even though HoundCI does not provide a badge, I admire your initiative of using a static badge. Good work.

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    }],
+    "require-jsdoc": ["error", {
+        "require": {
+            "FunctionDeclaration": true,
+            "MethodDefinition": true,
+            "ClassDeclaration": true
+        }
+    }]
+  }
+}
```
Comment (andela-moe) [inline .eslintrc]: This seems to be part of another PR. Any reason why it is here also?

**Review by andela-moe — COMMENTED** [review]
Please review comments and resolve where necessary.

File: .hound.yml (hunk 1 above)
Comment (nouwatinjacob) [reply .hound.yml]: New line added

File: README.md (hunk 2 above)
Comment (nouwatinjacob) [reply README.md:2]: Thanks
